File: C/RectangleArea.c
#include<stdio.h>

int main()
{
	int l, b, a, p;
	printf("Please enter length (in cms): ");
	scanf("%d", &l);
	printf("Please enter breadth (in cms): ");
	scanf("%d", &b);
	
	a = l * b;
	
	p = 2 * (l + b);
	
	printf("The area is %d sq. cms.\n", a);
	printf("The perimeter is %d cms.\n", p);
}

File: Java/RectangleArea.java
import java.util.Scanner;
class RectangleArea {
	public static void main(String[] args) {
		int l, b, a;
		Scanner scan = new Scanner(System.in);
		System.out.print("Enter length: ");
		l = scan.nextInt();
		System.out.print("Enter breadth: ");
		b = scan.nextInt();
		a = l * b;
		System.out.println("The area is " + a + " sq. units");
	}
}
